datafusion/expr/src/expr_fn.rs

```diff
@@ -697,7 +697,11 @@ pub fn interval_month_day_nano_lit(value: &str) -> Expr {
 /// # use datafusion_expr::test::function_stub::count;
 /// # use sqlparser::ast::NullTreatment;
 /// # use datafusion_expr::{ExprFunctionExt, lit, Expr, col};
-/// # use datafusion_expr::window_function::percent_rank;
+/// # use datafusion_expr::window_function::cume_dist;
+/// TODO
+/// how to handle import of percent_rank function from
+/// datafusion+functions_window
+/// as using this is causing the circular depency issue
 /// # // first_value is an aggregate function in another crate
```

Comment:
the expr function have written docs for the percent_rank,
but when I try to import the percent_rank from datafusion_functions_window workspace, 
its throwing error related to the circular depency 

how to handle such cases ?

Reply:
There are a couple of ways to get around this. But here I noticed that the `first_value` function is a mock and not the real `first_value` UDAF. 

So you can consider doing the same for `percent_rank`. Then you don't have to import anything. This is alright because the test is not testing the functionality of `percent_rank` in this doc test. 

Another advantage of mocking in this specific case is that the change remains local to the doc test which I believe is good.